src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfig.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeChange
```

Comment:
>CodeChange [](http://example.com/codeflow?start=17&length=10)

we usually have all classes in individual files, either way is fine , when you follow the same pattern - in this repo i still see individual files per class...

Reply:
fixed and moved https://github.com/dotnet/Scaffolding/pull/1593